test/test_retrieve_dns.py

```diff
@@ -29,6 +29,22 @@ def test_verify_invalid_dns(self):
             self.assertFalse(bin.retrieve_dns.verify_dn(dn),
                              "Invalid DN accepted: %s" % dn)
 
+    def test_verify_valid_service_types(self):
+        valid_service_types = ("gLite-APEL","uk.ac.gridpp.vcycle",
+                               "1.1234","_test")
+
+        for service_type in valid_service_types:
```

Comment:
is it right<sup>TM</sup> to pass each service type in turn to `bin.retrieve_dns.verify_service_types`? It looks like that method "expects" (or atleast can handle) a comma seperated list.